scrapy/extensions/memusage.py

```diff
@@ -75,7 +75,8 @@ def update(self):
         self.crawler.stats.max_value('memusage/max', self.get_virtual_size())
 
     def _check_limit(self):
-        if self.get_virtual_size() > self.limit:
+        current_mem_usage = self.get_virtual_size()
```

Comment:
similarly to the other comment, that's not the current memory usage, it's max memory usage